src/patternfly/sass-utilities/scss-variables.scss

```diff
@@ -151,6 +152,10 @@ $pf-global--spacer--form-element: pf-size-prem(6px) !default;
 $pf-global--gutter: $pf-global--spacer--md !default;
 $pf-global--gutter--md: $pf-global--spacer--lg !default;
 
+
+// Pade sidebar width
```

Comment:
"Page" 😄  
✨ for adding this as an offset

Reply:
😆